Doc/using/configure.rst

```diff
@@ -1396,12 +1396,21 @@ Compiler flags
 
    .. versionadded:: 3.2
 
+.. envvar:: CFLAGS_BUILTIN_MODULE
+
+   Equivalent flag to :envvar:`CFLAGS_NODIST` but it only applies to **all** builtin extension
+   modules built as part of the standard library.
+
+   Default: (empty).
+
+   .. versionadded:: 3.13
```

Comment:
Is it really new in 3.13.0? Use maybe "next"? Same remark for LDFLAGS_BUILTIN_MODULE.